### ISSUE TITLE
support Multiple invert-match?

### ISSUE THREAD

**Issue by zz5678** [BurntSushi/ripgrep#2627]
For the following to be processed:
```
abc
efg
123
```
my command is:
```
rg -v ab -v 12
```

This command doesn't actually work, but the ChatGPT answer works like this.

Is there a feature like this? But I'm using it in wrong way?

![Screenshot_20231013142128](https://github.com/BurntSushi/ripgrep/assets/67672527/778597c0-6971-41b7-8927-90fa12123808)

**Comment by s-p-turner** [comment]
Fwiw, two alternative ways to achieve what you want would be either:

rg -v ab "input-file" | rg -v 12

or (more efficient):

rg -v 'ab|12' "input-file"

**Comment by zz5678** [comment]
> Fwiw, two alternative ways to achieve what you want would be either:
> 
> rg -v ab "input-file" | rg -v 12
> 
> or (more efficient):
> 
> rg -v 'ab|12' "input-file"

Yes, the second method is more efficient, but it doesn't seem to appear in the man page of ripgrep

One question is, who told ChatGPT about the Way that doesn't exist????
